tests/unit/dataactcore/factories/staging.py

```diff
@@ -218,6 +218,59 @@ class Meta:
     uri = fuzzy.FuzzyText()
 
 
+class DetachedAwardFinancialAssistanceFactory(factory.Factory):
+    class Meta:
+        model = stagingModels.DetachedAwardFinancialAssistance
+
+    detached_award_financial_assistance_id = None
+    submission_id = fuzzy.FuzzyInteger(9999)
+    job_id = fuzzy.FuzzyInteger(9999)
+    row_number = fuzzy.FuzzyInteger(9999)
+    action_date = fuzzy.FuzzyText()
+    action_type = fuzzy.FuzzyText()
```

Comment:
I don't think we have that for the reimbursable fund columns that can only be 'D', 'R', or blank so we should be fine